### PR TITLE
Text modal

### PR DESCRIPTION
**Pull request by Fanuel-D**
Summary of Changes
• Implemented Map component: a component that renders a map of nearby study spots.
• Implemented Feedback component: based on whether or not a user is in the vicinity of a study spot, this asks users to give feedback on how busy the location is.

### CODE REVIEW

File: frontend/src/components/FeedbackForm.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    e.preventDefault();
+
+    axios
+      .post("http://localhost:3000/feedback", { userId, placeId, isBusy })
+      .then((response) => {
+        console.log("Feedback submitted:", response.data);
+      });
+  };
+
+  const isWithinRadius = (userLocation, placeLocation, radius) => {
```
Comment (Stannie-Lim) [inline frontend/src/components/FeedbackForm.jsx:33]: Maybe it'd be worthwhile to put all these helper functions into a `mapHelper.js` function then import them in here

File: frontend/src/components/Map.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  InfoWindow,
+} from "@react-google-maps/api";
+
+const containerStyle = {
+  width: "100%",
+  height: "600px",
+};
+
+function Map() {
+  const [location, setLocation] = useState({ lat: -3.745, lng: -38.523 });
```
Comment (Stannie-Lim) [inline frontend/src/components/Map.jsx:15]: Theoretically the center of the map should be the user's current location's lat and lng but this'll do for now (´・◡・｀)

Comment (Fanuel-D) [reply frontend/src/components/Map.jsx:15]: oh yeah that's just a place holder, I get the user's current location using the Geolocation API afterwards

File: frontend/src/components/FeedbackForm.jsx (hunk 1 above)
Comment (Fanuel-D) [reply frontend/src/components/FeedbackForm.jsx:33]: Resolved

**Pull request merged**